internal/ignition/ignition.go

```diff
@@ -125,6 +125,12 @@ allow chronyd_t container_file_t:sock_file write;
 allow chronyd_t spc_t:unix_dgram_socket sendto;
 `
 
+const agentFixBZ1964591 = `#!/usr/bin/sh
```

Comment:
One day I hope we will switch to use `go:embed` instead of putting these multi-line strings here

Reply:
Yeah, bash inside go is not the most elegant thing ever invented 😅